sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/implementation/EncryptionProcessor.java

```diff
@@ -339,6 +339,10 @@ public void encryptAndSerializeProperty(EncryptionSettings encryptionSettings, O
     }
 
     public Mono<byte[]> decrypt(byte[] input) {
+        if (input == null || input.length < 1) {
```

Comment:
nit:
`input.length < 1` seems a bit cryptic, also array len can never be negative, I would do `input.length == 0`

```suggestion
        if (input == null || input.length == 0) {
```

Reply:
done